brd-ios/breadwallet/Assets.xcassets/LogoBlue-Large.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
don't we have a large logo already in the assets?

Reply:
no, we don't have it

Reply:
LogoBlue is a vector and can be resized infinitely. you don't need a larger version of it :)

Reply:
okay then, updated :)